lab-maria/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "lab-02-tools-and-context",
+  "version": "1.0.0",
+  "description": "![cf](https://i.imgur.com/7v5ASc8.png) lab-02-tools-and-context ======",
+  "main": "index.js",
+  "scripts": {
+    "test": "mocha",
+    "lint": "eslint ./lib",
+    "start": "node ./lib/index.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/meklr/lab-02-tools-and-context.git"
+  },
+  "author": "maria leereed",
+  "license": "MIT",
+  "devDependencies": {
+    "eslint": "^3.19.0",
+    "mocha": "^3.2.0"
```

Comment:
remember to save your dev dependencies so when someone pulls down your repo, all they need to do is an npm install and they can run your app, as well as grab the dev dependencies required to test :)